app/controllers/application_controller.rb

```diff
@@ -10,4 +10,13 @@ def user_not_authorized
     flash[:alert] = 'Access denied'
     redirect_to request.referrer || root_path
   end
+  # Should be implemented in nearest future
```

Comment:
Delete commented code. You have git for version tracking, so you always could find things that where deleted.